### ISSUE TITLE
Waiting for QRCode Scan: Attempt 5

### ISSUE THREAD

**Issue by developerdevice** [orkestral/venom#908]
When I run Venom the following action occurs

> Waiting for QRCode Scan: Attempt 5

What I need is: Venom just attempts once, e.g "Waiting for QRCode Scan: Attempt 1" and closes after that. Is it possible?

My Code:

```
headless: true,
devtools: false,
useChrome: false,
debug: false,
logQR: false,
browserArgs: chromiumArgs,
refreshQR: 0,
autoClose: 0,
disableSpins: true,
```

**Comment by AlanMartines** [comment]
This problem has already been fixed [#907](https://github.com/orkestral/venom/commit/f21af7184964e28bc5cf0c4adbbfff2fa665c16a)

**Comment by AlanMartines** [comment]
Yes, it manipulates through **autoClose**.
